src/archivematicaCommon/lib/utilities/restart-elasticsearch-if-down

```diff
@@ -25,7 +25,7 @@ except pyes.exceptions.NoServerAvailable:
     subprocess.call(command_data)
 
     # note service was restarted in log
-    log_file = "/tmp/archivematica_elasticsearch_restarter.log"
+    log_file = "/var/log/archivematica/elasticsearch_restarter.log"
```

Comment:
Hm, I thought so too. Looks like it didn't get deleted from qa/1.x. I'll nuke it there.